<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="FilterTab">
       <attribute name="title">
        <string>Filtering</string>
       </attribute>
       <widget class="QGraphicsView" name="unfilteredGraph">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>10</y>
          <width>361</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="filteredGraph">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>240</y>
          <width>361</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="selectFileButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>93</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Select file</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>124</x>
          <y>26</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Selected:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
       <widget class="QLabel" name="selectedFileDisplay">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
